### PR TITLE
bug 1797601: force cert rotation every couple days for development

### PR DESCRIPTION
**Pull request by deads2k**
This will make DR easier to test and ensure rotation is working pretty well.

### CODE REVIEW

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 1]
```diff
@@ -119,6 +119,10 @@ func newCertRotationController(
 		klog.Warningf("Certificate rotation base set to %q", rotationDay)
 	}
 
+	// for the development cycle, make the rotation 60 times faster (every twelve hours or so).
+	// This must be reverted before we ship
+	rotationDay = rotationDay / 60
```
Comment (tnozicka) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]: should this be in an else block above or it break the configmap override?

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 2]
```diff
@@ -117,6 +117,10 @@ func newCertRotationController(
 		rotationDay = day
 		klog.Warningf("!!! UNSUPPORTED VALUE SET !!!")
 		klog.Warningf("Certificate rotation base set to %q", rotationDay)
+	} else {
+		// for the development cycle, make the rotation 60 times faster (every twelve hours or so).
+		// This must be reverted before we ship
```
Comment (xingxingxia) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]: Can we design it to have a way to set in install-config.yaml to pass the validity duration?

Comment (wangke19) [reply pkg/operator/certrotationcontroller/certrotationcontroller.go]: Agree with you, reducing the code change.

Comment (sttts) [reply pkg/operator/certrotationcontroller/certrotationcontroller.go]: We don't want to see this accessible in any way to customers without tainting the cluster as unsupported.

**Pull request merged**